### PR TITLE
SCRIPT-WEB-3935-SCRIPT-Feature: Introduce Localization Support

### PR DESCRIPTION
**Pull request by samwise-repo1**
SCRIPT-This PR introduces unit tests for the newly added string manipulation functions to ensure they work as expected.
 
 **PR Summary by Typo**
------------

**Overview**
This PR introduces localization support to the impress.js library, allowing the presentation to adapt to different languages.  The changes primarily involve modifications to the `samwise.js` file to handle language settings and update text content dynamically.

**Key Changes**
- Added functions to load and apply localized strings.
- Modified existing functions to use localized strings instead of hardcoded text.
- Introduced a new configuration option for specifying the language.
- Updated the toolbar button to display localized text.

**Recommendations**
Not deployment ready.  While the core localization functionality seems implemented, there are no tests to verify its correctness and robustness. Add comprehensive unit tests covering various language scenarios and edge cases to ensure production readiness.  Also, consider adding documentation on how to configure and use the localization feature.
 

 <h6>To turn off PR summary, please visit <a href="https://app.typoapp.io/settings/notification?tab=codeHealth">Notification settings</a>.</h6>

### CODE REVIEW

File: samwise.js [hunk 1]
```diff
@@ -4,20 +4,19 @@
  * Copyright 2016 Henrik Ingo, henrik.ingo@avoinelama.fi
```
Comment (frodo-repo) [inline samwise.js:4]: SCRIPT-The way you've refactored this complex method into simpler ones is great. It makes the code much more maintainable.
